Makefile

```diff
@@ -27,6 +27,7 @@ VERSION=$(shell cat ./VERSION)
 DOCKER_TAG=$(VERSION)-dev
 
 GOFLAGS=-ldflags "-X github.com/edgexfoundry/edgex-go.Version=$(VERSION)"
+GOTESTFLAGS?=-race
```

Comment:
This was added as a way to run go test on golang alpine based images. During the pipeline this set to an empty string via an env var thus disabling race condition testing on the sandbox. This has to be done until the project can switch to Go 1.15 where this has been fixed.